python/metaparticle_pkg/containerize.py

```diff
@@ -0,0 +1,75 @@
+import os
+import signal
+import sys 
+
+import metaparticle_pkg.option as option
+import metaparticle_pkg.builder as builder
+import metaparticle_pkg.runner as runner
+
+def is_in_docker_container():
+    mp_in_container = os.getenv('METAPARTICLE_IN_CONTAINER', None)
+    if mp_in_container in ['true', '1']:
+        return True
+    elif mp_in_container in ['false', '0']:
+        return False
+
+    try:
+        with open('/proc/1/cgroup', 'r+t') as f:
+            lines = f.read().splitlines()
+            last_line = lines[-1]
+            if 'docker' in last_line:
+                return True
+            elif 'kubepods' in last_line:
+                return True
+            else:
+                return False
+
+    except IOError:
+        return False
+
+
+def write_dockerfile(package, exec_file):
+    with open('Dockerfile', 'w+t') as f:
+        f.write("""FROM python:{version}-alpine
+
+COPY ./ /{name}/
```

Comment:
Are we going to install multiple packages in single docker image? 
I'd prefer to stick with hardoced (and always same) path like `/app/` if we don't need to install multiple packages.

Reply:
done.